.github/workflows/labeler.yml

```
@@ -10,3 +10,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/labeler@v5
+      - if: github.actor == 'dependabot[bot]'
```

Maybe not critical here but I heard that `github.actor` isn't too safe. https://www.synacktiv.com/publications/github-actions-exploitation-dependabot

```suggestion
      - if: github.event.pull_request.user.login == 'dependabot[bot]'
```